.github/dependabot.yml

```
@@ -11,3 +11,26 @@ updates:
   schedule:
     # Check for updates to GitHub Actions every week
     interval: "weekly"
+    
+- package-ecosystem: docker
+  directory: /cluster/images/
+  schedule:
+    interval: weekly
+
+- package-ecosystem: docker
+  directory: /cluster/images/
+  target-branch: "release-1.9"
+  schedule:
+    interval: monthly
```

```suggestion
    interval: weekly
```
How about also doing it weekly? 

I'm still thinking if we should run it by daily. Is there some side-effect if we shorten the interval?

---

just want to reduce the instability, we can run it with daily if we believe that the latest is always stable.

---

The reason why we bump the base image is usually regarding security, so i want to do it ASAP.
Given that we are using the latest `official` release, so I think we can trust it.

---

Weekly would be fine for me.

---

updated to Weekly